lib/kube/proxy/forwarder.go

```diff
@@ -786,6 +909,18 @@ func (f *Forwarder) setupContext(authCtx auth.Context, req *http.Request, isRemo
 // emitAuditEvent emits the audit event for a `kube.request` event if the session
 // requires audit events.
 func (f *Forwarder) emitAuditEvent(ctx *authContext, req *http.Request, sess *clusterSession, status int) {
+	_, span := f.cfg.tracer.Start(
+		req.Context(),
+		"kube/Forwarder.emitAuditEvent",
+		oteltrace.WithSpanKind(oteltrace.SpanKindServer),
+		oteltrace.WithAttributes(
+			semconv.RPCServiceKey.String(f.cfg.KubeServiceType),
+			semconv.RPCMethodKey.String("GlobalRequest"),
```

Comment:
What does `GlobalRequest` correspond to here and in similar calls to `tracer.Start`?